src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
         ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Bool -> Html Msg
+viewClaimModal loggedIn claim showClaimModal =
+    let
+        t =
+            loggedIn.shared.translators.t
+
+        greenTextTitleStyle =
+            "uppercase text-green text-xs"
+
+        claimVerifiersList =
+            "block mt-6 h-32 space-y-4"
+
+        photoAndTagName =
+            div []
+                [ Profile.viewLarge loggedIn.shared loggedIn.accountName claim.claimer ]
+
+        claimDateAndState =
+            let
+                date datetime =
+                    Just datetime
+                        |> Utils.posixDateTime
+                        |> Strftime.format "%d %b %Y" Time.utc
+
+                ( claimStatusPhrase, claimStatus, textColor ) =
+                    case claim.status of
+                        Approved ->
+                            ( t "claim.title_approved.1", t "claim.approved", "text-2xl font-bold lowercase text-green" )
+
+                        Rejected ->
+                            ( t "claim.title_rejected.1", t "claim.disapproved", "text-2xl font-bold lowercase text-red" )
+
+                        Pending ->
+                            ( t "claim.title_under_review.1", t "claim.pending", "text-2xl font-bold lowercase text-gray-600" )
```

Comment:
There is also a state where `claim.action.isCompleted`, where the claim's action is completed, but the claim is pending. In this state, the user and the reviewers haven't received any rewards, but the claim can't be voted on anymore (can't be approved or rejected). [Here's an example on the claimCard](https://github.com/cambiatus/frontend/blob/717c820a54df355498f257cb5efa33ffb363f8f3/src/elm/Claim.elm#L293-L306) that takes this case into account